l10n_eu_product_adr_dangerous_good/migrations/14.0.1.1.0/pre_migration.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 Camptocamp SA
```

Comment:
are you sure this file w/ this naming will be considered?

Reply:
After discussing with @sebalix I removed it anyway.
This module has never been merged, we just have to handle renaming for our customer.